src/db/QDjangoQuerySet.h

```diff
@@ -579,10 +580,11 @@ bool QDjangoQuerySet<T>::remove()
  *  executes its query.
  */
 template <class T>
-QDjangoQuerySet<T> QDjangoQuerySet<T>::selectRelated() const
+QDjangoQuerySet<T> QDjangoQuerySet<T>::selectRelated(QStringList *relatedFields) const
```

Comment:
Please document the new argument, explaining how it works.

Also, why a *pointer* to a QStringList instead of passing by reference? This seems un-Qt.